lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -261,6 +261,26 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
           currentValue: adoptOpenJreMatches.groups!.version,
         };
       }
+      const temurinJdkMatches = version.match(
+        /^temurin-(?<version>\d\S+)/
+      );
```

Comment:
@viceice I made it identical to the existing adoptopenjdk test. This could definitely be added, but then I would prefer adding it also to adoptopenjdk as they are otherwise equal implementations. (actually they could be merged to one regexp checking for temurin or adoptopenjdk) 🤔